src/JdbcUtils.java

```diff
@@ -7,7 +7,7 @@
 import java.util.Properties;
 
 public class JdbcUtils {
-    private static final String DB_PROPS = "db.properties";
+    private static final String DB_PROPS = "resources/db.properties";
```

Comment:
No, you should not specify resources directory as it is a default folder that will be scanned for your database properties. Remove `resources/` prefix